dev_ops/sql/Tables/CreateContracts.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE contracts (
+    id BIGSERIAL PRIMARY KEY,
+	proposer_id BIGINT REFERENCES users (id),
+	streamer_id BIGINT REFERENCES users (id),
+	game TEXT NOT NULL,
+	description TEXT NOT NULL,
```

Comment:
same here, would like to see a varchar with a reasonable limit instead of text

Reply:
Going to keep this one as TEXT because we don't know how long descriptions will be.